packages/react/src/components/Forms/Checkbox/index.spec.tsx

```diff
@@ -106,4 +112,13 @@ describe("<Checkbox/>", () => {
       document.querySelector(".c__checkbox__group.c__field.c__field--error"),
     ).toBeInTheDocument();
   });
+  it("renders with text and text right", async () => {
```

Comment:
Why not putting the line `expect(spyError).not.toHaveBeenCalled();` inside this test? I'm a bit worried about having a standalone `it(...)` that has a scope that sets expectations about things outside itself.